geoips_clavrx/plugins/yaml/product_inputs/abi.yaml

```diff
@@ -0,0 +1,25 @@
+abi:
```

Comment:
@jsolbrig Should we just approve this as-is, then go back through with your current round of Issues/PRs to fix the naming / formatting of these product YAMLs?

Reply:
Yes, move now, rename in another PR.